docs: move history and collections features to dedicated page (#1073)
acdh-oeaw/apis-core-rdf: docs/source/features.rst

Additional Features
===================

APIS is designed to be extensible and customizable. This section describes some 
additional features that are shipped with APIS and how to use them.

History plugin
--------------

The :py:mod:`apis_core.history` module provides versioning 

> to the APIS framework. It is based on the `django-simple-history`_

Reviewer: ```suggestion
for APIS. It is based on the `django-simple-history`_
```

package.

VersionMixin
^^^^^^^^^^^^

The :class:`apis_core.history.models.VersionMixin` class is a mixin
that can be added to any model to enable versioning. It adds a `history`
property to the model that returns a `HistoricalRecords` instance. Additionally
it allows to override the date of the revision by setting the `_history_date`
property of the model instance.
To activate versioning for a model, simply inherit from `VersionMixin`:

.. code-block:: python

    from django.db import models
    from apis_core.apis_history.models import VersionMixin

    class MyModel(VersionMixin, models.Model):
        pass

API endpoint
^^^^^^^^^^^^

The :class:`apis_core.history.api_views.GenericHistoryLog` class is a viewset
that provides a REST API endpoint for the version history of a model. It can be
used to retrieve the version history of a model instance.

> The viewset can be accessed under `/apis/api/history/entity/edit_log/`. It takes

Reviewer: I don't find `edit_log` anywhere in the codebase, I guess that has changed?

Author: good catch, changed the url to the one that is actually used
